docker/README.md

```diff
@@ -31,3 +31,76 @@ docker build -f docker/base.Dockerfile . -t bridge-base:latest
 ```sh
 docker build -f docker/rt.Dockerfile . -t bridge-rt:latest
 ```
+
+## Running locally
+
+### Pre-requisites
```

Comment:
Can we add something between headers so that we don't have empty space between them?